README.md

```diff
@@ -1,26 +1,113 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
 
-Things you may want to cover:
+> A mobile web application where you can manage your budget
 
-* Ruby version
+Home page |
+----------|
+![Screenshot from 2021-12-04 13-28-14](https://user-images.githubusercontent.com/65068771/144707791-ac4b41f3-2af1-4945-8c65-627dc30c6fca.png)
 
-* System dependencies
+  
 
-* Configuration
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Database creation
+## Built With
 
-* Database initialization
+- Ruby on Rails
+- PostgreSQL
 
-* How to run the test suite
+## Video description
+[Loom video](https://loom.com/share/9fb0fa00b7c147ad928599dbcdf16334)
```

Comment:
- [ ] In the video presentation, please highlight a piece of the code you find interesting or are proud of making.